### PR TITLE
relay service

### PR DESCRIPTION
**Pull request by pelikhan**

### CODE REVIEW

File: services/relay.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Relay
+
+    identifier: 0x183fe656
+
+A switching relay.
+
+## Registers
+
+    rw active: bool @ intensity
```
Comment (pelikhan) [inline services/relay.md:9]: should this be value?

Comment (mmoskal) [reply services/relay.md:9]: No. Maybe cal it "conducting" ?

Comment (pelikhan) [reply services/relay.md:9]: "close" ?

Comment (mmoskal) [reply services/relay.md:9]: I think "closed" may be interpreted the other way by people not familiar with electric circuits; conducting should be clear (if they know the word)

Comment (pelikhan) [reply services/relay.md]: conductive it is

**Comment by mmoskal** [comment]
you could also add registers for maximum current and voltage

**Review by mmoskal — APPROVED** [review]

File: services/relay.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    event on @ active
+
+Emitted when relay goes from ``off`` to ``on`` state.
+
+    event off @ inactive
+
+Emitted when relay goes from ``on`` to ``off`` state.
+
+    const max_switching_current?: u16 A @ 0x180
```
Comment (mmoskal) [inline services/relay.md:32]: this register is under events; I guess we should have the spectool check for that
also I think u16.16 would be better here

**Pull request merged**